packages/create-sitecore-jss/src/templates/nextjs/next.config.js

```diff
@@ -53,7 +53,12 @@ const nextConfig = {
       {
         source: '/healthz',
         destination: '/api/healthz',
-      }
+      },
+      // rewrite for default not-found page 
```

Comment:
I'm thinking, in general, can it be applied only to the not-found page? Maybe there are more cases and we need to provide a general description and as an example - not found page?

Reply:
Agree, comment could prob be simply "service endpoints"